src/app/explore/meta-campaign-banner/meta-campaign-banner.component.scss

```diff
@@ -9,31 +11,72 @@ div.banner {
     position: absolute;
     max-width: 2000px;
     width: 100%;
-    height: 100%;
+    height: 400px;
     left: 50%;
     transform: translateX(-50%);
+    bottom: 0;
+    display: none;
+    @media ("min-width: 450px") {
+      display: block;
+      bottom: initial;
+      height: 100%;
+    }
   }
 }
 
 div.sleeve {
   display: flex;
-  align-items: center;
+  flex-direction: column;
+  align-items: start;
   justify-content: flex-start;
   max-width: 1200px;
-  height: 600px;
   margin-left: auto;
   margin-right: auto;
-  padding: 0 20px;
+  padding: 0;
   box-sizing: border-box;
+  height: 100%;
+
+  @media #{abstract.$breakpoint-smallish} {
```

Comment:
I'm a bit confused about using this var for one change but also a similar but distinct breakpoint (480px vs 450px) through the rest of this diff.

Could we pick one size and use a var consistently? I think it's quite helpful to be able to see the app's common breakpoints of interest in one place. I think it would also be fine to redefine `$breakpoint-smallish` to 480px for simplicity; it would just mean that a small number of additional mobile sizes might see the 'Donation to...' label on the donate form over 2 lines.

Reply:
Cool, yeah that sounds good, I don't think it's very obvious that they're deliberately couple right now.